DESCRIPTION

```diff
@@ -43,6 +41,4 @@ Suggests:
 SystemRequirements: GNU make
 Language: en-US
 VignetteBuilder: knitr
-Remotes: 
-    mathematica-mpr/coda
```

Comment:
How do we feel about this?

Reply:
How is coda used within BCF? And is there a non-Mathematica coda?

Reply:
There is a non-MPR coda its a generic MCMC utils package. 

- https://cran.r-project.org/web/packages/coda/index.html

I believe Dan T. was involved in updating to the MPR version, I might see if he has an opinion about this

Reply:
The issue with the non-MPR coda is it completely ignores cross-chain disagreement which gives crazy neffs when the model isn't converged e.g. 906 vs. 25 for one of our TCPI runs. So if one chain says tau = 1000 +/-10 and the other says tau = 0 +/-10, coda will say chain 1 has low autocorr so neff_2=ndraws, chain 2 has low autocorr so neff_1=ndraws, neff = neff_1+neff_2, so neff = ndraws yay! whereas stan goes no, neff=2 since they don't mix. 
Looks like the newly-updated coda has the same way of calculating neff, so it's still a problem. 

That said having our parallel universe coda was in retrospect a terrible idea.